app/scripts/lib/app-start.js

```diff
@@ -33,14 +33,14 @@ define(function (require, exports, module) {
   const HeightObserver = require('lib/height-observer');
   const IframeChannel = require('lib/channels/iframe');
   const InterTabChannel = require('lib/channels/inter-tab');
-  const MarketingEmailClient = require('lib/marketing-email-client');
+  /*const MarketingEmailClient = require('lib/marketing-email-client');*/
```

Comment:
All the commented out code needs to be deleted 👍

Reply:
Sure @vladikoff